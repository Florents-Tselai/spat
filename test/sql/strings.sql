SELECT set_val_in_shmem(1236);
SELECT get_val_in_shmem();
SELECT spat_set('k1', 'myval');
SELECT spat_set('k1', 'myval');
SELECT spat_set('k2', 'myval');
SELECT store_size();
