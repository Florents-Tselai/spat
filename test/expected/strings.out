SELECT set_val_in_shmem(1236);
 set_val_in_shmem 
------------------
 
(1 row)

SELECT get_val_in_shmem();
 get_val_in_shmem 
------------------
             1236
(1 row)

SELECT spat_set('k1', 'myval');
 spat_set 
----------
        1
(1 row)

SELECT spat_set('k1', 'myval');
 spat_set 
----------
        1
(1 row)

SELECT spat_set('k2', 'myval');
 spat_set 
----------
        2
(1 row)

SELECT store_size();
 store_size 
------------
          2
(1 row)

